ansible/roles/mailcow/defaults/main.yml

```diff
@@ -8,12 +12,13 @@ mc_git_repo: https://github.com/mailcow/mailcow-dockerized
 # Specific version of mailcow git repository; commit, branch name or tag
 mc_git_version: master
 
-mc_min_memory: 1500 # in MB
+# RAM in MB
+mc_min_memory: 1500
 mc_ip: "{{ ansible_default_ipv4.address }}"
 mc_install_dir: /opt/mailcow-dockerized
 mc_dbname: mailcow
-mc_dbpass: "{{ lookup('password', '/dev/null length=68 chars=ascii_letters,digits') }}"
```

Comment:
Why was `/dev/null` removed?

Reply:
This was removed to fix idempotency.

From [Ansible documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/password_lookup.html)
>A special case is using /dev/null as a path. The password lookup will generate a new random password each time, but will not write it to /dev/null. This can be used when you need a password without storing it on the controller.